private-bucket/versions.tf

```diff
@@ -0,0 +1,9 @@
+
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    aws = {
+      source = "hashicorp/aws"
```

Comment:
does it make sense to add a provider version here? or is this general enough that it works with any version?

Reply:
Thank you :flushed: I put in the documentation for this repo that all modules should have provider versions, and then immediately forgot to do it